### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by meghasalunke1**
Updated an example in step 6 https://www.ibm.com/docs/en/cloud-paks/cloud-pak-watson-aiops/3.6.1?topic=appliances-installing-aws#configuring-amazon-ec2-to-import-the-appliance

### CODE REVIEW

**Comment by meghasalunke1** [comment]
@miq-bot add_label bug

**Comment by meghasalunke1** [comment]
@miq-bot add-label oparin/yes?

**Comment by meghasalunke1** [comment]
@miq-bot add-label master/yes?

**Comment by meghasalunke1** [comment]
@miq-bot assign @Fryguy

**Comment by meghasalunke1** [comment]
@Fryguy This should go in both master and oparin releases. Thanks!

File: installing_on_aws/_topics/installation.md [hunk 1]
```diff
@@ -106,9 +106,10 @@ These are the procedural steps as of the time of writing. For the latest informa
                "s3:GetBucketLocation",
                "s3:GetObject",
                "s3:ListBucket",
-               "s3:PutObject"
+               "s3:PutObject",
+               "s3:GetBucketAcl"
              ],
-             "Resource": ["arn:aws:s3:::BUCKET_TO_UPLOAD_IMAGE","arn:aws:s3:::BUCKET_TO_UPLOAD_IMAGE/*"]
+             "Resource": ["arn:aws:s3:::wt-cp4mcm-bcdr","arn:aws:s3:::wt-cp4mcm-bcdr/*"]
```
[the remark this answers is not in the document]
Comment (meghasalunke1) [reply installing_on_aws/_topics/installation.md]: Done

**Comment by Fryguy** [comment]
Backported to `oparin` in commit ebb7d3ddc2c643c7a33a85a23c56c93ac82128a3.

```text
commit ebb7d3ddc2c643c7a33a85a23c56c93ac82128a3
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed Feb 15 16:49:24 2023 -0500

    Merge pull request #1721 from meghasalunke1/Upated-example-file-for-Configuring-amazon-EC2
    
    Update installation.md
    
    (cherry picked from commit e6d9e51fb814921882973ed9ce9111ad43c25bf9)
```

**Pull request merged**